lib/linguist/heuristics.rb

```diff
@@ -491,5 +499,14 @@ def call(data)
         Language["XML"]
       end
     end
+
+    disambiguate ".user" do |data|
```

Comment:
Yep, superb idea. Let's pretend it was never brought up. 😝 

LGTM. /cc @lildude for final review